tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -300,6 +300,13 @@ When('the user renames file/folder {string} to an invalid name {string} using th
   return client.page.FilesPageElement.filesList().renameFile(fromName, toName, false)
 })
 
+When('the user tries to rename file/folder {string} to {string} using the webUI', function(
```

Comment:
same as the step above, why do we need both?

Reply:
I wanted to ping you actually in chat if I could drop the previous one and use this one instead. I'd like this change because of the tests where the rename is not expected to go through but at the same time the name is still valid (e.g. if the resource was deleted before the rename). In that case, step saying specifically "to an invalid name" doesn't feel right. On the other hand, I don't see a problem with having the "tries to rename" also on invalid names tests.

Would you agree with this change?

Reply:
having only the "tries" step sounds good for me, get rid of the other one